mne_bids/tests/test_write.py

```diff
@@ -699,6 +699,24 @@ def test_fif_ias(tmpdir):
     assert raw.info['chs'][0]['kind'] == FIFF.FIFFV_IAS_CH
 
 
+@pytest.mark.filterwarnings(warning_str['channel_unit_changed'])
+def test_fif_exci(tmpdir):
+    """Test writing FIF files with excitation channel."""
+    data_path = testing.data_path()
+    raw_fname = op.join(data_path, 'MEG', 'sample',
+                        'sample_audvis_trunc_raw.fif')
+    raw = _read_raw_fif(raw_fname)
+
+    match = r'The unit for channel\(s\) MEG 0113 has changed'
+    with pytest.warns(RuntimeWarning, match=match):
+        raw.set_channel_types({raw.ch_names[0]: 'exci'})
```

Comment:
I think this part isn't needed anymore now, right @hoechenberger ?

```suggestion
    raw.set_channel_types({raw.ch_names[0]: 'exci'})
```